articles/migrations/index.md

```diff
@@ -25,6 +25,102 @@ If you need help with the migration, create a ticket in our [Support Center](${e
 
 Current migrations are listed below, newest first. For migrations that have already been enabled see [Past Migrations](#past-migrations).
 
+### Summary of Endpoint Migrations
+
+This table is a summary of the below endpoint migrations for the first part of 2018. See the entries below the table for more detailed explanations.
+
```

Comment:
I found this matrix a bit confusing. To start with, I would use the same date format as the migrations do.

Either `Deadline` and `April 01, 2018`
or `Mandatory Opt-In` and `2018-04-01`

I also expected a `more info` link that would take me to the relevant migration notice in this doc.